types/html-escaper/package.json

```diff
@@ -2,6 +2,10 @@
     "private": true,
     "name": "@types/html-escaper",
     "version": "3.0.9999",
+    "exports": {
+        "import": "./index.d.mts",
```

Comment:
I wouldn't do it like this' instead change to `type=module`, then put the ESM in `index.d.ts`.

Reply:
If doing this, we will get another error.

> @arethetypeswrong/cli
> html-escaper v3.0.3
> @types/html-escaper v3.0.9999
> 
> Build tools:
> - rollup@^2.39.0
> 
>  (ignoring rules: 'no-resolution', 'cjs-only-exports-default', 'unexpected-module-syntax', 'cjs-resolves-to-esm')
> 
> 👺 Import resolved to an ESM type declaration file, but a CommonJS JavaScript file. https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/FalseESM.md

It is because the package itself has put a `exports` field in its `package.json`, as indicated by the source code in [npm page](https://www.npmjs.com/package/html-escaper?activeTab=code). Hence both esm and cjs typing need to be added.

Maybe my PR title is not clear enough. Sorry about that.

Reply:
Sorry, I meant my comment as a diff from this PR, which is to say "add type=module, then move your existing mts file to the main d.ts file".

Reply:
Done.